### PR TITLE
refactor: restructure repo

### PR DESCRIPTION
**Pull request by Favna**

### CODE REVIEW

**Review by kyranet — CHANGES_REQUESTED** [review]
Tests need to be fixed.

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# The MIT License (MIT)
+
+Copyright © `2019` `Antonio Román`
```
Comment (cfanoulis) [inline LICENSE.md:3]: not only kyra, there's vlad as well (btf org)

**Review by cfanoulis — CHANGES_REQUESTED** [review]
some stuff

**Review by cfanoulis — APPROVED** [review]
el gee tee em

**Review by kyranet — APPROVED** [review]

**Pull request merged**